bridges/otelzap/core_test.go

```diff
@@ -93,3 +94,66 @@ func TestConvertLevel(t *testing.T) {
 		}
 	}
 }
+
+// Copied from field_test.go. https://github.com/uber-go/zap/blob/b39f8b6b6a44d8371a87610be50cce58eeeaabcb/zapcore/memory_encoder_test.go
```

Comment:
What is copied from `field_test.go`?

Should it not be?

```suggestion
// Copied from https://github.com/uber-go/zap/blob/b39f8b6b6a44d8371a87610be50cce58eeeaabcb/zapcore/memory_encoder_test.go.
```

Reply:
ooh yes right